notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -13,6 +13,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    fig, _ = plot_adv_visit_sweep(df, df2, "$\\texttt{Latest}$")
</pre><p>Not really relevant to the review, but it's interesting that win rate used to improve with adversary visits but is now basically flat. I guess this is just reflecting that either the victim fools for even the simple trick, or won't be fooled by something even a little more sophisticated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>